src/test/kotlin/pt/up/fe/ni/website/backend/controller/AccountControllerTest.kt

```diff
@@ -142,6 +142,35 @@ class AccountControllerTest @Autowired constructor(
                 jsonPath("$.websites[0].iconPath") { value(testAccount.websites[0].iconPath) }
             }
         }
+        @Test
+        fun `should create an account with a empty website list`() {
+            val noWebsite = Account(
+                "Test Account",
+                "no_website@email.com",
+                "test_password",
+                "This is a test account",
+                TestUtils.createDate(2001, Calendar.JULY, 28),
+                "https://test-photo.com",
+                "https://linkedin.com",
+                "https://github.com"
+            )
+
+            mockMvc.post("/accounts/new") {
+                contentType = MediaType.APPLICATION_JSON
+                content = noWebsite.toJson()
```

Comment:
This doesn't actually test what you want. `Account` has an empty list of websites so `noWebsite.toJson()` will also have an empty list, instead of a missing value. The best way to test this might be by using `mapOf()` or `AccountDto` for example